packages/@okta/vuepress-site/docs/reference/api/admin-notifications /index.md

```diff
@@ -0,0 +1,893 @@
+---
```

Comment:
nit: there seems to be an extra whitespace and the end of the filename

Reply:
oh no. nice catch, thanks.